лабораторная 4/task1.py

```diff
@@ -0,0 +1,119 @@
+if __name__ == "__main__":
+    # Write your solution here
+    class Vehicle:
+        """
+        Базовый класс для всех транспортных средств.
+
+        Атрибуты:
+            make (str): Марка транспортного средства.
+            model (str): Модель транспортного средства.
+            year (int): Год выпуска.
+            speed (float): Текущая скорость транспортного средства.
+        """
+
+        def init(self, make: str, model: str, year: int, speed: float = 0.0):
```

Comment:
```suggestion
        def __init__(self, make: str, model: str, year: int, speed: float = 0.0):
```
Метод инициализации экземпляров класса, а также магические методы всегда начинаются и заканчиваются двумя нижними подчеркиваниями